src/dfi/govvariables/attributes.h

```diff
@@ -588,8 +588,8 @@ class ATTRIBUTES : public GovVariable, public AutoRegistrator<GovVariable, ATTRI
     static const std::map<std::string, uint8_t> &allowedVaultIDs();
     static const std::map<std::string, uint8_t> &allowedRulesIDs();
     static const std::map<uint8_t, std::map<std::string, uint8_t>> &allowedKeys();
-    static const std::map<uint8_t, std::map<uint8_t, std::function<ResVal<CAttributeValue>(const std::string &)>>>
-        &parseValue();
+    static const std::map<uint8_t, std::map<uint8_t, std::function<ResVal<CAttributeValue>(const std::string &)>>> &
```

Comment:
Is this how the new clang format looks?

Reply:
I have no idea 🥲

Reply:
Weird. Possible. Not sure if there's a config.

Reply:
clang format does push it to the next line :)

Reply:
https://github.com/DeFiCh/ain/pull/2566/commits/fd0fcd488610bbba4267c52a7dcc5a9eb40ff9d4